src/ApiService/ApiService/onefuzzlib/Events.cs

```diff
@@ -89,7 +89,7 @@ public class Events : IEvents {
 
         public virtual void LogEvent(BaseEvent anEvent) {
             var serializedEvent = JsonSerializer.Serialize(anEvent, anEvent.GetType(), _options);
-            _log.LogInformation("sending event: {EventType} - {serializedEvent}", anEvent.GetEventType(), serializedEvent);
+            _log.LogInformation("logging event: {EventType} - {serializedEvent}", anEvent.GetEventType(), serializedEvent);
```

Comment:
That was meant to describe what is happening on line 86.

Reply:
Ah I see, I think we could remove `LogEvent` since it's only used by `SendEvent`. It'll make things a little clearer

Reply:
I agree but I think this function is used elsewhere. If that is the case, I suggest you take the log message string (or just a prefix) as parameter to this function. That wat you keep the intent close to where it is being used.